### DISCUSSION TITLE
Can each and every user in Grafana rearrange and resize panels in Grafana and save.

### DISCUSSION THREAD

**Discussion by a-imran99** [grafana/grafana#58027] [Dashboards, Panels & Visualizations]
Can all users rearrange and resize panels and save dashboards and when a particular user logs in, they should be able to see their own dashboard and not a common dashboard.

This means each and every user should view a different version of dashboards.

@torkelo It would be great if you suggest a way :) , Thank you.

**Comment by a-imran99** [comment]
Is different dashboards for different user possible? I mean every user can have their own set of dashboards which they can only customize on structural level and cannot edit a panel. @torkelo ?

**Reply by a-imran99** [reply]
https://github.com/grafana/grafana/discussions/44690

Attaching my previous query.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
